RELEASING.md

```diff
@@ -0,0 +1,8 @@
+# Release Process
+
+1. Add release entry to [changelog](./CHANGELOG.md)
+2. Add new release notes to the Changelog.
```

Comment:
Hmm. Multiple sources of RELEASING content has caused me to dupe the CHANGELOG update step. 🤭

Reply:
Removed